### PR TITLE
Update Catalan translation.

### PR DESCRIPTION
**Pull request by jmallach**
This is a full Catalan translation update for Keycloak 26.

### CODE REVIEW

**Comment by jmallach** [comment]
I kindly request @ECron's review as in #22879.

**Comment by jmallach** [comment]
Sorry, in my initial comment I mentioned Keycloak 25 but in fact the translation was done against master before it was branched to 26, ie the target for this PR is master AND release-26.

**Comment by ssilvert** [comment]
@jmallach We need an issue to be linked to this PR.  Can you create one and link it as per the contribution guidelines?

**Comment by jonkoops** [comment]
@ssilvert we actually do not require any issues to be linked for localization updates up to now. The only requirement is that this PR is reviewed by another native speaker of the language.

**Comment by jmallach** [comment]
@jonkoops I'm waiting for @Ecron to have a look, he just told me via Telegram he'll have a look soonish.

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties
Comment (Ecron) [inline js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties:35]: Al final cal substituir la coma simple per les cometes llatines (»).

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties
Comment (Ecron) [inline js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties:36]: Falta una cometa llatina de tancament al final de la variable {{0}}.

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties
Comment (Ecron) [inline js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties:67]: Potser falta una cometa llatina de tancament al final de la variable {{0}}.

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties
Comment (Ecron) [inline js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties:84]: "no válid" millor que "invàlid".

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties
Comment (Ecron) [inline js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties:141]: Observació: «Privacitat» o «Privadesa»? (qüestió d'estil).

**Comment by ahus1** [comment]
@jmallach / @Ecron - Please keep the discussions in English as much as possible. I know discussing translations might be an edge case, still it allows others to follow the discussion more easily.

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties
Comment (Ecron) [inline js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties:201]: Potser «codis nous» millor? (qüestió d'estil)

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties
Comment (Ecron) [inline js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties:213]: Millor «permès» si anem amb la varietat central.

File: themes/src/main/resources-community/theme/base/admin/messages/messages_ca.properties
Comment (Ecron) [inline themes/src/main/resources-community/theme/base/admin/messages/messages_ca.properties:64]: Potser s'ha d'escapar l'apòstrof?

File: themes/src/main/resources-community/theme/base/email/messages/messages_ca.properties
Comment (Ecron) [inline themes/src/main/resources-community/theme/base/email/messages/messages_ca.properties:53]: «Heu estat convidats» o «Heu estat convidat»?

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties
Comment (jmallach) [reply js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties:141]: I'll switch to privadesa here and in another appearance, but termcat says they are interchangeable.

File: themes/src/main/resources-community/theme/base/login/messages/messages_ca.properties
Comment (Ecron) [inline themes/src/main/resources-community/theme/base/login/messages/messages_ca.properties:309]: Faltaria un article: «... amb el proveïdor...» o «... amb un proveïdor...».

File: themes/src/main/resources-community/theme/base/login/messages/messages_ca.properties
Comment (Ecron) [inline themes/src/main/resources-community/theme/base/login/messages/messages_ca.properties:462]: Potser no cal escapar l'apòstrof.

File: themes/src/main/resources-community/theme/base/login/messages/messages_ca.properties
Comment (Ecron) [inline themes/src/main/resources-community/theme/base/login/messages/messages_ca.properties:471]: Potser no cal escapar l'apòstrof. A les següents línies passa el mateix, hi ha barreja d'apòstrofs escapats i no escapats.

**Review by Ecron — COMMENTED** [review]
Reviewed. Some minor comments and fixes in some strings.

File: themes/src/main/resources-community/theme/base/admin/messages/messages_ca.properties
Comment (jmallach) [reply themes/src/main/resources-community/theme/base/admin/messages/messages_ca.properties:64]: I don't think so, but will look what other teams did.

**Comment by jmallach** [comment]
@ssilvert @jonkoops from my side, things are ready, I have applied the corrections suggested by @Ecron (not all, some were suggestions). Many thanks for the review!

**Comment by jonkoops** [comment]
Great! Please resolve the conversations that have been processed so we can see the status. @Ecron could you approve the PR if it looks good to you like this?

**Comment by jmallach** [comment]
@jonkoops as my main interest was the 26 release, will this appear in that branch as is, or do I need a new PR with the same content, targetting the release branch that references this one?

**Comment by jonkoops** [comment]
Everything that gets merged into `main` will become part of the v26 release. So once this PR is merged it will end up there.

**Review by Ecron — APPROVED** [review]

**Review by jonkoops — APPROVED** [review]

**Comment by jonkoops** [comment]
Thanks all!

**Pull request merged**
